rllib/core/models/specs/checker.py

```diff
@@ -200,18 +190,23 @@ def check_input_specs(
 
     Examples (See more examples in ../tests/test_specs_dict.py):
 
-        >>> class MyModel(nn.Module):
-        ...     @property
-        ...     def input_specs(self):
-        ...         return {"obs": TensorSpec("b, d", d=64)}
-        ...
-        ...     @check_input_specs("input_specs")
-        ...     def forward(self, input_data, return_loss=False):
-        ...         ...
+    .. testcode::
+        import torch
+        from torch import nn
+        from ray.rllib.core.models.specs.specs_base import TensorSpec
 
-        >>> model = MyModel()
-        >>> model.forward({"obs": torch.randn(32, 64)}) # No error
-        >>> model.forward({"obs": torch.randn(32, 32)}) # raises ValueError
+        class MyModel(nn.Module):
+            @property
+            def input_specs(self):
+                return {"obs": TensorSpec("b, d", d=64)}
+
+            @check_input_specs("input_specs")
+            def forward(self, input_data, return_loss=False):
+                ...
+
+        model = MyModel()
+        model.forward({"obs": torch.randn(32, 64)}) # No error
+        model.forward({"obs": torch.randn(32, 32)}) # raises ValueError
```

Comment:
This will cause CI to fail. You could put this in a subsequent testcode that's explicitly skipped

```
    model.forward({"obs": torch.randn(32, 64)}) # No error

... and this example raises a ValueError

.. testcode::
    :skipif: True

    model.forward({"obs": torch.randn(32, 32)}) 

.. testoutput::
    
    Traceback:
        ...
```